### PR TITLE
Add "Join Us" page

### PR DESCRIPTION
**Pull request by bahar-ap**
## Description
This PR demonstrates the changes made to website repo for adding a permanent Join Us page instead of temp ones around recruiting time.
There can still be more improvements on this piece of code as mentioned below, however I think this PR has the main structure of the page (I also have attached the sketch of initial design if that helps)
As it can be seen on commits, I have installed  react-bootstrap package so could use Progress bar and Accordion which has made some further changes to other packages and files.

## Some further actions could be the following: 
* fix the spacing, margin and styling pretty much
* could improve on colours esp for progress bar 
* could have a react progress steps instead - check the following links I found (however with reactstrap progress steps I couldnt find how to put labels on them so I chose progress bars)
    https://codesandbox.io/examples/package/react-step-progress-bar
    https://snyk.io/advisor/npm-package/react-step-progress-bar

## Screen Shots:
image_from_ios

## Steps to Test
`yarn develop` and you will see the changes

### CODE REVIEW

**Review by kdav108 — COMMENTED** [review]
LGTM! If okay we won't go live with it yet, need to just add links to the 'Apply to Rider Dev' and 'Apply to other teams' buttons. I'll try to source them asap, but not going to tie you down to this PR while I get them.

**Comment by kdav108** [comment]
Could someone review this PR and give me a thumbs up :)

File: package.json
Comment (Blake-Haydon) [inline package.json:15]: Upgrading from bootstrap v4 to v5 requires updating some components as many of the class names have changed. I was hesitant to increase the version due to this. It shouldn't be too hard to fix however, just need to read through the breaking changes and ctrl-f through this repo.

### See incorrect formating of pictures when compaired to the current live version [here](https://monashhumanpower.org/team)
image

**Review by Blake-Haydon — COMMENTED** [review]
The accordion and progress bar looks fantastic! However, the upgrade to the bootstrap version has broken other parts of the site.

File: package.json
Comment (kdav108) [reply package.json:15]: Ahhh good catch, but really annoying to find out this :(

**Comment by kdav108** [comment]
For now, I have reverted back to bootstrap 4.5. If we upgraded, we would need to do more testing to check which sections of the current UI has broken, which may take quite long and hold off this feature unnecessarily. I'll add this as a backlog card.

Thanks @Blake-Haydon for pointing this out.

**Review by Blake-Haydon — APPROVED** [review]
LGTM! Thanks for reverting back bootstrap @kdav108.

**Comment by hallgchris** [comment]
Looks good, though I'd suggest
- Running the preview by Ops before going live
- Mentioning explicitly that rider dev is always recruiting. This isn't made clear on the site

**Comment by megascrapper** [comment]
Looks like a good start. Definitely an improvement over just a google form link. 

Looking at other team's recruitment page, I think it'd be great if we include some description for each recruitment, as well as some EOI form when recruitments are closed.

Here's what I'm currently working on:
image

(ignore the blurry image, idk what I did there)

There's a switch in `joinUs.md` called `is_open` which is set to true for rider and false for general recruitment. When recruitments are closed, a different description is shown and EOI button is shown instead of actual recruitment form.

**Comment by kdav108** [comment]
@megascrapper That's actually looking quite good, especially because it follows the same format as other pages and addresses the concerns raised by @hallgchris.
We currently don't have an EOI, can we make it so if the EOI link in joinUs.md is empty, we don't display the EOI button.

File: src/markdown/joinUs.md
Comment (kdav108) [inline src/markdown/joinUs.md:4]: Could we move this under the general recruitment category (Rider dev application is always open, so EOI is only applicable for general recruitment.

**Review by kdav108 — COMMENTED** [review]
Looking really good!

Just a few tiny suggestions, can we rename 'Recruitment Process' to 'Our Recruitment Process', I feel it flows much better, what do u think?

Also, for the first FAQ, can we mention that Rider dev recruitment is open all year while general recruitment occurs every 6 months. 

I know you didn't initially develop the above 2, but would be good if u can fix them. Also just run through this website with Harry and Em perhaps, and then we should be good to go live!

**Comment by kdav108** [comment]
Whoops one more thing I forgot, we need to rebase this branch with latest code in master, this branch has become quite old :)

**Comment by megascrapper** [comment]
> Whoops one more thing I forgot, we need to rebase this branch with latest code in master, this branch has become quite old :)

I've merged all latest changes from master.

**Comment by kdav108** [comment]
@megascrapper I have updated a few descriptions, just test it again form your end and then we should be good to merge this in :)

**Pull request merged**
